core/src/main/java/nu/yona/server/messaging/entities/MessageDestination.java

```diff
@@ -5,21 +5,16 @@
 package nu.yona.server.messaging.entities;
```

Comment:
- I think caching ALL messages is too much overhead.
- No you can delete using a query on the destination column.
- The extra overhead is: pulling the `MessageDestination` entity from the database by ID, and pushing the `MessageDestination` entity again.